nzedb/http/Capabilities.php

```diff
@@ -69,30 +70,28 @@ public function output($data, $params, $xml = true, $type = '')
 	{
 		$this->type = $type;
 
+		$options = [
+			'Parameters' => $params,
+			'Data'       => $data,
+			'Server'     => $this->getForMenu(),
+			'Type'       => $type
+		];
+
+		$response = (new XML_Response($options))->returnXML();
+
```

Comment:
The JSON response is now dependent upon the XMLWriter response, so it is needed.  This way we only need to manage one class for arranging the data.  We take a small performance hit because we then have to load the XML data and convert it to JSON if JSON was requested.